### DISCUSSION TITLE
Missing packages when installing

### DISCUSSION THREAD

**Discussion by yubing5021** [perspective-dev/perspective#2237] [Q&A]
I am trying to install perspective into pycharm however I am getting an issue where it looks like I am missing a couple of packages during installation: as I am getting the below error msgs:

Package 'perspective.extension' is absent from the `packages` configuration.
Package 'perspective.include.perspective' is absent from the `packages` configuration.
'perspective.include.perspective.python' is absent from the packages configuration
Package 'perspective.labextension' is absent from the `packages` configuration.
Package 'perspective.labextension.static' is absent from the `packages` configuration.
Package 'perspective.nbextension.static' is absent from the `packages` configuration.
Package 'perspective.src' is absent from the `packages` configuration.
Package 'perspective.tests.single_threaded' is absent from the `packages` configuration.

am I missing a step? figured when entering the pip install perspective-python would have the package included. included source code below.

[Error Code.txt](https://github.com/finos/perspective/files/11582392/Error.Code.txt)

**Comment by texodus** [comment]
>  ModuleNotFoundError: No module named 'cmake'

... which you're trying to build from source because your python version is incorrect (3.11)

**Reply by yubing5021** [reply]
i have cmake installed, what version should i have?

**Comment by timkpaine** [comment]
None of those are errors, you do not have cmake installed as "cmake --version" command does not run. Ensure cmake is installed (the application, not the python wrapper) and follow the developer guide. Alternatively, use a distribution with prebuilt 3.11 binaries like conda or switch to a python version where we already distribute binaries (3.7-3.10).
